frontend/isoko/src/components/BusinessPreview.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Rating } from 'react-simple-star-rating';
+import { Table } from 'react-bootstrap';
+import MinorityTag from './MinorityTag';
+import KeywordTag from './KeywordTag';
+
+const BusinessContainer = styled(Table)`
+   width: 85%;
+   height: 25%;
+   background-color: #fff;
+   border: 1px solid #cecece;
+   box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+   min-height: 20vh;
+   min-width: 60vh;
+   margin-bottom: 3%;
+`;
+
+const Photo = styled.img`
+   height: 100%;
+   width: 100%;
+   object-fit: cover;
+`;
+
+const RightContainer = styled(Table)`
+   justify-content: flex-start;
+   display: flex;
+   flex-grow: 1;
+   flex-direction: column;
+   padding-left: 0.25em;
+`;
+
+const Text = styled.div`
+   font-size: 0.75rem;
+   display: flex;
+   flex-shrink: 1;
+   padding-bottom: 0.5em;
```

Comment:
I think if the description is too long or the size of the screen is too small, it will start to look weird.   If we dont want to cut off text, and instead have it go to a new line, try playing around with the  white-space: initial;  css property. 

Another solution is we could cut off the text and add a ... if it gets cut off. You can do that using a few of the properties that are on here https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow